### PR TITLE
Certains des problèmes principaux sont corrigés

### PR DESCRIPTION
**Pull request by liZe**

### CODE REVIEW

File: gibolt/routes.py [hunk 1]
```diff
@@ -50,10 +52,10 @@ def authorized(oauth_token):
         return abort(403)
     session["token"] = oauth_token
     session["user_id"] = github.get("/user", access_token=oauth_token)["id"]
-    session["repository_names"] = {
+    session["repository_names"] = [
```
Comment (gentooboontoo) [inline gibolt/routes.py:55]: Out of curiosity, why this change?

File: gibolt/templates/circle/item_delete.html.jinja2 [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{% extends '_layout.jinja2' %}
+
+{% block content %}
+  <h2>
+    Suppression de
+    {% if item.item_type == 'checklist' %}
+      l'action récurente {{ item.content }}
```
Comment (gentooboontoo) [inline gibolt/templates/circle/item_delete.html.jinja2]: _récurrente_ #multiple

File: gibolt/static/sass/style.sass [hunk 3]
```diff
@@ -232,3 +259,15 @@ main
   padding-right: 0.75em
   opacity: 0.7
 
+.delete-milestone
+  float: right
+  font-size: 0;
```
Comment (gentooboontoo) [inline gibolt/static/sass/style.sass]: Superfluous semicolon?

**Review by gentooboontoo — COMMENTED** [review]
First review. I will fix typos.

File: gibolt/static/sass/style.sass (hunk 3 above)
Comment (liZe) [reply gibolt/static/sass/style.sass]: Oops :smile:

File: gibolt/templates/circle/item_delete.html.jinja2 [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{% extends '_layout.jinja2' %}
+
+{% block content %}
+  <h2>
+    Suppression de
+    {% if item.item_type == 'checklist' %}
+      l'action récurente {{ item.content }} 
+    {% else %}
+      l'indicateur {{ item.cotent }}
```
Comment (gentooboontoo) [inline gibolt/templates/circle/item_delete.html.jinja2]: co+n+tent

File: gibolt/routes.py (hunk 1 above)
Comment (liZe) [reply gibolt/routes.py:55]: Because sets are not (by default) JSON-serializable and can’t be stored in sessions.

Comment (gentooboontoo) [reply gibolt/routes.py:55]: Thanks, seems legit :wink:

File: gibolt/templates/circle/item_delete.html.jinja2 [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{% extends '_layout.jinja2' %}
+
+{% block content %}
+  <h2>
+    Suppression de
+    {% if item.item_type == 'checklist' %}
+      l'action récurrente <em>{{ item.content }}</em>
```
Comment (gentooboontoo) [inline gibolt/templates/circle/item_delete.html.jinja2:7]: Emphasis added here.
